### PR TITLE
Overlay for Lenovo Z5s(L78071)

### PR DESCRIPTION
**Pull request by ghost**
worked：
- auto brightness
- notch/status_bar
- brightness change

### CODE REVIEW

**Comment by phhusson** [comment]
Looks good to me. @tadi777 ?
Should we remove values-land?

**Comment by TadiT7** [comment]
> Looks good to me. @TadiT7 ?
> Should we remove values-land?

Here, yup, it's a Pie device, also the height should be changed to px

File: Lenovo/Z5s/res/values/notch.xml [hunk 1]
```diff
@@ -17,9 +17,7 @@
 */
 -->
 <resources>
-    <dimen name="rounded_corner_radius">30dip</dimen>
-    <dimen name="status_bar_height">30.0dip</dimen>
-    <dimen name="status_bar_height_landscape">30.0dip</dimen>
-    <dimen name="status_bar_height_portrait">30.0dip</dimen>
+    <dimen name="rounded_corner_radius">30dp</dimen>
```
Comment (TadiT7) [inline Lenovo/Z5s/res/values/notch.xml]: Is it a device with 480 dpi by default? Then that would translate to 90px instead, as rounded corner radius is physical.

Comment (pantanshin) [reply Lenovo/Z5s/res/values/notch.xml]: Ye, thank you,i have modified

**Comment by mrgreywater** [comment]
I don't think those build files are supposed to be in this PR, right?

```
build/aapt_mac 
build/build_mac.sh 
build/libc++.dylib 
build/signapk/libconscrypt_openjdk_jni.dylib 
build/signapk/signapk_mac.jar 
```

File: Lenovo/Z5s/res/values/notch.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+** Unless required by applicable law or agreed to in writing, software
+** distributed under the License is distributed on an "AS IS" BASIS,
+** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+** See the License for the specific language governing permissions and
+** limitations under the License.
+*/
+-->
+<resources>
+    <dimen name="rounded_corner_radius">90px</dimen>
+    <dimen name="status_bar_height">87px</dimen>
```
Comment (mrgreywater) [inline Lenovo/Z5s/res/values/notch.xml]: `status_bar_height_landscape` and `status_bar_height_portrait` seems to be missing

File: Lenovo/Z5s/res/values/notch.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    <string translatable="false" name="config_mainBuiltInDisplayCutout">M -80,0 L -50,90 L 50,90 L 80,0 Z</string>
```
Comment (mrgreywater) [inline Lenovo/Z5s/res/values/notch.xml]: I don't actually know how to interpret that path, but I find it curious that it only has coordinates divisible by 10, but the `status_bar_height` is not divisible by 10. Shouldn't the cutout height be equal to `status_bar_height`?

**Comment by pantanshin** [comment]
> I don't think those build files are supposed to be in this PR, right?
> 
> ```
> build/aapt_mac 
> build/build_mac.sh 
> build/libc++.dylib 
> build/signapk/libconscrypt_openjdk_jni.dylib 
> build/signapk/signapk_mac.jar 
> ```

Maybe， 
Just thinking that other people using macOS can build

**Pull request closed without merging**